api/libretime_api/storage/models/file.py

```diff
@@ -28,7 +28,14 @@ class ImportStatus(models.IntegerChoices):
     is_hidden = models.BooleanField(db_column="hidden", blank=True, null=True)
 
     is_accessed = models.IntegerField(db_column="currentlyaccessing")
-    edited_by = models.ForeignKey(
+    is_scheduled = models.BooleanField(db_column="is_scheduled", blank=True, null=True)
+    is_part_of = models.BooleanField(db_column="is_playlist", blank=True, null=True)
```

Comment:
Maybe is_playlisted? Is part of implies that the value is the object it is a part of

Reply:
It can also be a block, maybe `is_part_of_list` ? Maybe `is_playlisted` is good enough, hope the spell checker doesn't scream !

Reply:
`is_part_of_list` would work